doc/apps/s_client.pod

```diff
@@ -420,7 +420,7 @@ command for more information.
 send the protocol-specific message(s) to switch to TLS for communication.
 B<protocol> is a keyword for the intended protocol.  Currently, the only
 supported keywords are "smtp", "pop3", "imap", "ftp", "xmpp", "xmpp-server",
-"irc" and "postgres."
+"irc", "postgres", and "ldap".
```

Comment:
punctuation should come inside the quotes.  since you're hear, please fix it in 422 and 423.